cf_units/tests/integration/parse/test_graph.py

```diff
@@ -3,9 +3,6 @@
 # This file is part of cf-units and is released under the BSD license.
 # See LICENSE in the root of the repository for full licensing details.
 # ruff: noqa: E402
-import pytest
-
-antlr4 = pytest.importorskip("antlr4")
```

Comment:
Reverts the change from #423. cc @ocefpaf - the idea here is to vendor antlr4 runtime so that we don't have to worry about it as a dependency.